api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -46,6 +51,15 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
+        String locationAttributeUuid = Context.getAdministrationService().getGlobalProperty("locationbasedaccess.locationAttributeUuid");
+        PersonAttributeType personAttributeType = new PersonAttributeType();
+        if ( locationAttributeUuid != null && locationAttributeUuid.trim() != "") {
+            personAttributeType.setUuid(locationAttributeUuid);
+        }
+        personAttributeType.setName(this.LOCATION_ATTRIBUTE_TYPE_NAME);
```

Comment:
Does the above line compile?
I this first create a separate pull request and set up travis

Reply:
Yes @dkayiwa  I was able to compile in my local instance. 
Please have a look at this PR - #6 for Travis